target/linux/ath79/image/generic.mk

```diff
@@ -999,6 +999,21 @@ define Device/librerouter_librerouter-v1
 endef
 TARGET_DEVICES += librerouter_librerouter-v1
 
+define Device/meraki_mr16
+  SOC := ar7161
+  DEVICE_VENDOR := Meraki
+  DEVICE_MODEL := MR16
+  ROOTFS_SIZE := 13440k
+  IMAGE_SIZE := 15680k
+  DEVICE_PACKAGES := kmod-spi-gpio kmod-owl-loader
```

Comment:
I'm looking at kmod-spi-gpio for the first time, do we need it here? Most probably that's already enabled elsewhere, otherwise one of the other devices would require it as well ...

Reply:
Removed. Doesn't seem necessary, no.